app/views/events/index.html.erb

```diff
@@ -0,0 +1,67 @@
+<h3 class="text-center">List of All Events</h3>
+<div class="container">
+<table class="table">
+  <thead>
+    <tr>
+      <th>Event name</th>
+      <th>Description</th>
+      <th>Event Date</th>
+      <th>Event Creator</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr><% @events.each do |e| %></tr>
+      <td><%= e.title %></td>
+      <td><%= e.description %></td>
+      <td><%= e.event_date %></td>
+      <td><%= e.creator.username %></td>
+      <td><%= link_to "Show details", event_path(e) %></td>
+    <% end %>
+ </tbody>
+</table>
+</div>
+<div class="container">
+  <h3 class="text-center">All upcoming Events</h3>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Event name</th>
+        <th>Description</th>
+        <th>Event Date</th>
+        <th>Event Creator</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr><% @all_upcoming_events.each do |e| %></tr>
+        <td><%= e.title %></td>
+        <td><%= e.description %></td>
+        <td><%= e.event_date %></td>
+        <td><%= e.creator.username %></td>
+        <td><%= link_to "Show details", event_path(e) %></td>
+      <% end %>
```

Comment:
## Remove all the logic related code 

* [ ] View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)
* [ ] There shoud be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* Check this article on [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code) and [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more info about this. Thank you. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright.